server/routes/shared_api/autocomplete/autocomplete.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+from urllib.parse import urlencode
+
+from flask import Blueprint
+from flask import request
+
+from server.routes.shared_api.autocomplete import helpers
+from server.routes.shared_api.place import findplacedcid
+
+# TODO(gmechali): Add Stat Var search.
+
+# Define blueprint
+bp = Blueprint("autocomplete", __name__, url_prefix='/api')
+
+
+@bp.route('/autocomplete', methods=['GET', 'POST'])
```

Comment:
I'd use just GET here since the request is read-only and idempotent.

Reply:
Done.